README.md

```diff
@@ -1,4 +1,32 @@
-# sarif-pattern-matcher
+<p align="center"><img src="Src/img/header_img.png" width="440" /></p>
 
-Quality domain agnostic regular expression pattern matcher that persists results
-to SARIF
+<p align="center">
+
+  <a href="https://github.com/microsoft/sarif-pattern-matcher">
```

Comment:
each link should point to a different place, for example, if we are showing release, we should link to release, if we show packages, so, nuget.org

Reply:
Thanks for that point sure i can refract that